texas-hearing-institute/listening_settings/ListeningButtonGroup.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { Pressable, View, Text, StyleSheet } from 'react-native';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export default function ListeningButtonGroup() {
+    const [selectedMode, setSelectedMode] = useState("");
+
+    useEffect(() => {
```

Comment:
we should avoid using `useEffect` when we're not trying to sync with some external system ([read more here](https://react.dev/learn/you-might-not-need-an-effect)). we just want to check the `asyncStorage` for an initial value, so we can fetch this inside the initial `useState` - example:
![image](https://github.com/rice-apps/texas-hearing-institute/assets/71530873/4ec95a93-c238-42e6-95ad-c4b291906ad7)